<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pricingForm</class>
 <widget class="QWidget" name="pricingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>106</y>
     <width>781</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>3</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>قیمت تمام شده</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="primePrice">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>8</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
border-radius:10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>1000000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sellPrice">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>63</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
border-radius:10px;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <double>-1000000000000000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="Btn" name="FoodselectPB">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>5</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>منو</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/img/input.png</normaloff>:/img/input.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodName">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);
border-radius:10px;</string>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>انتخاب غذا</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>0</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Nazanin</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>قیمت تراز</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveReplacePB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>486</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>مقادیر جایگزین شوند</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveNewNamePB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>486</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>با نام جدید ذخیره شود</string>
   </property>
  </widget>
  <widget class="Btn" name="retrieveDataPB">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>7</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>بازگردانی مقادیر اولیه</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/img/rewind.png</normaloff>:/img/rewind.png</iconset>
   </property>
  </widget>
  <widget class="Btn" name="calculatePB">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>50</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>محاسبه</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/img/calculate.png</normaloff>:/img/calculate.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>479</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(85, 170, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tableView</zorder>
  <zorder>label_2</zorder>
  <zorder>primePrice</zorder>
  <zorder>sellPrice</zorder>
  <zorder>FoodselectPB</zorder>
  <zorder>foodName</zorder>
  <zorder>retrieveDataPB</zorder>
  <zorder>calculatePB</zorder>
  <zorder>frame_2</zorder>
  <zorder>saveReplacePB</zorder>
  <zorder>saveNewNamePB</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Btn</class>
   <extends>QPushButton</extends>
   <header>MaterialsForm.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="reso.qrc"/>
 </resources>
 <connections/>
</ui>
